src/disclosures/js/views/financial-view.js

```diff
@@ -619,6 +624,19 @@ var financialView = {
   },
 
   /**
+   * Sets visibility of graduation cohort content. Hidden if not available.
+   * @param {boolean} visibility - Whether or not a graduation cohort
+   * was provided
+   */
+  graduationCohortVisible: function( visibility ) {
+    if ( visibility === false ) {
```

Comment:
Trivial: prefer leading with true values in if conditionals because they're a bit easier to parse when reading the code. This is a straightforward example but could possibly be improved slightly with:

```
if ( visibility === true ) {
   this.$graduationCohortContent.show();
} else {
   this.$graduationCohortContent.hide();
}
```

Alternatively, if you update the visibility variable name, it could be shortened to:

```
if ( isVisible ) {
   this.$graduationCohortContent.show();
} else {
   this.$graduationCohortContent.hide();
}
```